### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Shegsdev**
I uninstalled the templating engine and rearrange my folder structure. I also implemented the MVC pattern for modularization and maintenance. The currently written API endpoints are working as expected.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -91,35 +42,9 @@ app.get('/api/v1/meetups/upcoming', (req, res) => {
...
-	res.status(200).send({
-		status: 200,
-		data: meetups,
-	});
-});
-
 
 // Upvote a specific question
-app.patch('/api/v1/questions/:id/upvote', (req, res) => {
+app.patch('/questions/:id/upvote', (req, res) => {
```
Comment (saheedt) [inline app.js]: remove all route instance from your app.js / server.js. All api routes should be in the api route file.

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
```
Comment (saheedt) [inline .eslintrc]: install the airbnb rules `eslint-config-airbnb-base` module via `npm` and add this to your eslint rules.

```
 "extends": "airbnb-base"
```

**Review by saheedt — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (Shegsdev) [reply app.js]: Fixed.
https://github.com/Shegsdev/Questioner/pull/18

File: .eslintrc (hunk 2 above)
Comment (Shegsdev) [reply .eslintrc]:
```
    "rules": {...
    },
    "extends": "airbnb-base"
```
updated .eslintrc file

**Comment by Shegsdev** [comment]
Ready to merge.

**Pull request merged**